### PR TITLE
improve gke project-setup doc

### LINKED ISSUE #2121: Error running through GCP Project Setup
https://www.kubeflow.org/docs/gke/deploy/project-setup/

> 7 Initialize your project to ready it for Anthos Service Mesh installation.

```
curl --request POST \
  --header "Authorization: Bearer $(gcloud auth print-access-token)" \
  --data '' \
  https://meshconfig.googleapis.com/v1alpha1/projects/${PROJECT_ID}:initialize
```

> Refer to Anthos Service Mesh documentation for details.
> You do not need a running GKE cluster. The deployment process creates a cluster for you.

Executing this step generates the following error:

```
{
  "error": {
    "code": 400,
    "message": "Workload Identity Pool does not exist (project-xxx.svc.id.goog).",
    "status": "INVALID_ARGUMENT"
  }
}
```

I've verified that the project exists and the meshconfig api is enabled for it.

### PR DESCRIPTION
**Pull request by theofpa**
Fixes #2121

### CODE REVIEW

**Comment by 8bitmp3** [comment]
Hi @theofpa Thanks!

/assign @joeliedtke @Bobgy 

What's your view?

File: content/en/docs/gke/deploy/project-setup.md [hunk 1]
```diff
@@ -76,6 +76,10 @@ Follow these steps to set up your GCP project:
 
     Refer to [Anthos Service Mesh documentation](https://cloud.google.com/service-mesh/docs/archive/1.4/docs/gke-install-new-cluster#setting_credentials_and_permissions) for details.
 
+    For more information, see the following issues:
+
+    * [kubeflow/website #2121](https://github.com/kubeflow/website/issues/2121) Projects which have never had a k8s cluster before might need to create and delete a temp cluster to activate this.
```
Comment (Bobgy) [inline content/en/docs/gke/deploy/project-setup.md:81]: Nit: avoiding acronym like k8s and temp make it easier to read

**Review by Bobgy — CHANGES_REQUESTED** [review]

File: content/en/docs/gke/deploy/project-setup.md (hunk 1 above)
Comment (8bitmp3) [inline content/en/docs/gke/deploy/project-setup.md:81]: @theofpa Thank you again. What do you think of this? Rephrased with more context:

```suggestion
    If you encounter a `Workload Identity Pool does not exist` error, refer to the following issue:

    * [kubeflow/website #2121](https://github.com/kubeflow/website/issues/2121)
    describes that creating and then removing a temporary Kubernetes cluster may
    be needed for projects that haven't had a cluster set up beforehand.
```

**Review by 8bitmp3 — CHANGES_REQUESTED** [review]
@theofpa Thank you again. What do you think of this? Rephrased with more context:

cc @Bobgy 

```diff
-     For more information, see the following issues:

-       * [kubeflow/website #2121](https://github.com/kubeflow/website/issues/2121) Projects which have never had a k8s cluster before might need to create and delete a temp cluster to activate this.

+     If you encounter a `Workload Identity Pool does not exist` error, refer to the following issue:

+     * [kubeflow/website #2121](https://github.com/kubeflow/website/issues/2121)
+     describes that creating and then removing a temporary Kubernetes cluster may
+     be needed for projects that haven't had a cluster set up beforehand.
```

File: content/en/docs/gke/deploy/project-setup.md (hunk 1 above)
Comment (theofpa) [reply content/en/docs/gke/deploy/project-setup.md:81]: Much better, thanks @8bitmp3

Comment (8bitmp3) [reply content/en/docs/gke/deploy/project-setup.md:81]: No problem @theofpa If you liked that revision, please implement it and we'll LGTM and approve it asap.

**Comment by Bobgy** [comment]
/lgtm
/approve
Thank you!

**Pull request merged**